sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -484,7 +494,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
     }
     this._tokenRenewalTimer = setTimeout(async () => {
       try {
-        await this._negotiateClaim(true);
+        await this._negotiateClaim(true, operationOptions);
```

Comment:
Looks like all the negotiateClaim usages require passing in both arguments _anyways_ so we can make both non-optional, similar to what I mentioned for _ensureTokenRenewal. Also, rather than passing undefined for `setTokenRenewal` we can pass in false.

Reply:
Yes